data/json/bionics.json

```diff
@@ -650,25 +645,26 @@
     "id": "bio_lighter",
     "type": "bionic",
     "name": { "str": "Finger Lighter" },
-    "description": "The index fingers of both hands have powerful fire starters which extend from the tip.",
-    "occupied_bodyparts": [ [ "hand_l", 1 ], [ "hand_r", 1 ] ],
+    "description": "The index finger of your left hand has fire starter which extend from the tip.",
+    "occupied_bodyparts": [ [ "hand_l", 1 ] ],
     "fake_item": "fake_firestarter",
-    "act_cost": "3 kJ"
+    "act_cost": "1500 J"
   },
   {
     "id": "bio_lockpick",
     "type": "bionic",
     "name": { "str": "Fingerpick" },
     "description": "One of your fingers has an electronic lockpick surgically embedded in it.  This automatic system will quickly unlock all but the most advanced key locks without any skill required on the part of the user.",
-    "occupied_bodyparts": [ [ "hand_r", 2 ] ],
+    "occupied_bodyparts": [ [ "hand_l", 2 ] ],
     "act_cost": "50 J"
   },
   {
     "id": "bio_magnet",
     "type": "bionic",
     "name": { "str": "Electromagnetic Unit" },
-    "description": "Surgically embedded in your right hand is a powerful electromagnet, allowing you to use your own strength to pull all nearby magnetic objects towards you.  Unlucky bystanders might be injured or killed by flying objects.",
-    "occupied_bodyparts": [ [ "hand_r", 3 ] ],
+    "description": "Surgically embedded in your left arm is a powerful electromagnet, allowing you to use your own strength to pull all nearby magnetic objects towards you.  Unlucky bystanders might be injured or killed by flying objects.",
+    "occupied_bodyparts": [ [ "arm_l", 9 ] ],
+    "encumbrance": [ [ "arm_l", 3 ] ],
     "act_cost": "10 kJ"
```

Comment:
This should probably be higher.

Reply:
The CBM itself is 0.7kg and it fits in your arm. (Audit to set vol/weight based on used slots?)
So I assume it's some super "space zombies" tech on par with Aftershock top tier items.
The forearm doesn't flex either. So only weight would cause encumbrance, as it would be some really dense sinter like current neodymium ones

Reply:
Those are permanent magnets; this is an electromagnet. I'd remove the encumbrance but add to the kJ.

Reply:
Removed enc, boosted power consumption to 15, still within borders estimated in #34456